submitted_models/explorer_r3_sensor_config_1/model.sdf

```diff
@@ -226,7 +242,7 @@
               </ray>
       </sensor>
       <velocity_decay/>
-      <sensor name="rs_back" type="rgbd_camera">
+      <sensor name="rs_rear" type="rgbd_camera">
```

Comment:
This line should be reverted to `rs_back` or the instances of `rs_back` in `vehicle_topics.launch` will need to be updated to `rs_rear`.

Reply:
I have changed all rs_back to rs_rear now.